### ISSUE TITLE
Bug: multiple instances of babel-polyfill

### ISSUE THREAD

**Issue by samit4me** [SSENSE/vue-carousel#254]
Just updated from `0.9.0` to `0.10.0` and when running my project I get the following error:

```
vue-carousel.min.js?fe42:6 Uncaught Error: only one instance of babel-polyfill is allowed
    at Object.eval (vue-carousel.min.js?fe42:6)
    at Object.eval (vue-carousel.min.js?fe42:6)
    at n (vue-carousel.min.js?fe42:6)
    at Object.eval (vue-carousel.min.js?fe42:6)
    at n (vue-carousel.min.js?fe42:6)
    at Object.eval (vue-carousel.min.js?fe42:6)
    at n (vue-carousel.min.js?fe42:6)
    at Object.eval (vue-carousel.min.js?fe42:6)
    at n (vue-carousel.min.js?fe42:6)
    at eval (vue-carousel.min.js?fe42:6)
```

Going to stick with `0.9.0` for now but was wondering if you have any recommendations? If it's of any help we `import 'babel-polyfill';` at the beginning of `main.js` and using webpack 3.

**Comment by brianmclachlin** [comment]
Having the same issue, downgrading to `0.9.0` as well.

If the package used the following code it would fix the issue:

```
if (!global._babelPolyfill) {
	require('babel-polyfill');
}
```

**Comment by quinnlangille** [comment]
Hey @brianmclachlin, I actually just noticed this myself. Seems like there was an oversight in review and we ended up merging in two 'babel-polyfill' declarations. Will resolve in the next release!

**Comment by stephwestgarth** [comment]
Just flagging this one again as a new release went out yesterday but the issue wasn't fixed. Still having to use 0.9.0 because of this issue.

**Comment by Suruat** [comment]
+1 to the issue. Can't use plugin because of this error

**Comment by kjugi** [comment]
#266 fixing this issue I think

**Comment by quinnlangille** [comment]
Yeah @kjugi is right, I've cleared my calendar for this afternoon and will (hopefully) be merging + releasing it in this afternoon

**Comment by quinnlangille** [comment]
Forgot to update that this was released yesterday with `v0.12.0`! Everything looks good on my end, but feel free to comment here if anyone is still experiencing babel-polyfill related issues :octocat:
